test/distance/edge-filters-test.cpp

```diff
@@ -0,0 +1,115 @@
+#include <gtest/gtest.h>
+
+#include "test/common/common.hpp"
+#include "src/distance/edge.hpp"
+#include "src/distance/edge-filters.hpp"
+#include "src/providers/factory.hpp"
+
+namespace found {
+
```

Comment:
As a general note on test conventions

1. Don't use underscores in the test names
2. Only 1 new line between test cases